python/python/tests/test_async_client.py

```diff
@@ -4047,6 +4047,31 @@ async def test_setbit(self, redis_client: TRedisClient):
         with pytest.raises(RequestError):
             await redis_client.setbit(set_key, 0, 0)
 
+    @pytest.mark.parametrize("cluster_mode", [True, False])
+    @pytest.mark.parametrize("protocol", [ProtocolVersion.RESP2, ProtocolVersion.RESP3])
+    async def test_getbit(self, redis_client: TRedisClient):
+        key = get_random_string(10)
+        non_existing_key = get_random_string(10)
+        set_key = get_random_string(10)
+        value = "foobar"
+
+        assert await redis_client.set(key, value) == OK
+        assert await redis_client.getbit(key, 1) == 1
+        assert await redis_client.getbit(key, 1000) == 0
```

Comment:
please add a comment - it's not immediately obvious to anyone why this would return 0 unless they read the command documentation.